tests/gnrc_dhcpv6_client/tests-as-root/01-run.py

```diff
@@ -15,6 +15,11 @@
 
 
 IA_NA_ADDRESS_POOL_PREFIX = "2001:db8:1::"
+IA_PD_PREFIX = "2001:db8:8000::"
+
+
+def assert_unique_elements(*args):
+    assert len(args) == len(set(args))
```

Comment:
This does not need to be a function and makes it unnecessarily harder to trace the actual spot the error happened. Rather just say e.g.

```py
assert global_addr_1 != global_addr_2
```

below.

Reply:
I was actually thinking that there might be a third test case (e. g. IA_TA) added later which could be more easily tested this way than making individual assertions. But you right that this makes things unnecessarily complicated. I changed it in one of the fixup commits.